### ISSUE TITLE
Autocomplete not working properly in aliases - version 0.8.0

### ISSUE THREAD

**Issue by Everettss** [graphql/graphiql#222]
GraphiQL autocomplete shows wrong hints when used in aliased field.
This problem doesn't occurs in version `0.7.0`.

My code look like this

```javascript
import graphqlHTTP from 'express-graphql';
import {
    GraphQLSchema,
    GraphQLObjectType,
    GraphQLString
} from 'graphql';

const Schema = new GraphQLSchema({
    query: new GraphQLObjectType({
        name: 'Query',
        fields: {
            article: {
                type: new GraphQLObjectType({
                    name: 'Article',
                    fields: {
                        title: { type: GraphQLString },
                        content: { type: GraphQLString }
                    }
                }),
                resolve() {
                    return {
                        title: 'Lorem ipsum',
                        content: 'Sed ut perspiciatis'
                    };
                },
            },
        },
    }),
});

app.use('/graphql', graphqlHTTP({
    schema: Schema,
    graphiql: true
}));
```
### This is how it looks like in `graphiql`

 1. Root autocomplete works properly

<img width="342" alt="screen shot 2016-11-19 at 20 39 12" src="https://cloud.githubusercontent.com/assets/8555194/20458180/e8cfda56-ae9d-11e6-9a86-edb6b2f45b95.png">

 2. Article autocomplete works properly

<img width="344" alt="screen shot 2016-11-19 at 20 39 33" src="https://cloud.githubusercontent.com/assets/8555194/20458181/eacea36e-ae9d-11e6-8374-302658c3f2ba.png">

 3. **Autocomplete on aliased article shows root fields! - it's broken.**

<img width="346" alt="screen shot 2016-11-19 at 20 51 15" src="https://cloud.githubusercontent.com/assets/8555194/20458182/ec454e3c-ae9d-11e6-82fe-7ee145f53a86.png">

### Version of packages

    "express-graphql": "0.6.1",
    "graphql": "0.8.2",

`graphiql` is served from CDN: `http://cdn.jsdelivr.net/graphiql/0.8.0/graphiql.min.js`

**Comment by asiandrummer** [comment]
Thanks for the report @Everettss - this is a known issue and we're working to resolve it as soon as possible.

**Comment by asiandrummer** [comment]
Fixed under [v0.5.9](https://github.com/graphql/codemirror-graphql/commit/63aad40f3d46f6cc8872db6ead97f72408cba99f) release. `graphiql` updated to reflect the changes in [9157a7](https://github.com/graphql/graphiql/commit/9157a7ffa8ff3848fff69c2a725a092fe93d7813).

Thanks for the report!
